.github/workflows/cloud_csharp_client_test.yaml

```diff
@@ -0,0 +1,100 @@
+name: Hazelcast cloud test with csharp client
+
+on:
+  workflow_dispatch:
+    inputs:
+      organization_name:
+        description: Default is hazelcast, but if you would like to run the workflow with your forked repo, set your github username
+        required: true
+        default: hazelcast
+      branch_name:
+        description: Name of the branch to test client from
+        required: true
+        default: master
+      base_url:
+        description: Base url of the cloud env. I.e https://uat.hazelcast.cloud
+        required: true
+        default: https://uat.hazelcast.cloud
+      hzVersion:
+        description: Version of hazelcast (For cloud standard cluster, it has different versions, i.e 5.0.2-2)
+        required: true
+
+jobs:
+  test_standard_cluster:
+    runs-on: ubuntu-latest
+    name: Hazelcast cloud standard cluster tests with csharp
+    steps:
+      - name: Checkout to scripts
+        uses: actions/checkout@v2
+        
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+          
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+          
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: |
+            3.1.x
+            5.0.x
+            6.0.x
```

Comment:
Do we need to run these tests with those versions? I mean, wouldn't one of them be enough?

Reply:
Yes, we need to run it for all. We have support for all those versions. Time to time there may differences in framework versions which is handled separately.

Reply:
But as far as I can see, we only use 5.0 to run the tests 
```
dotnet test bin/Release/net5.0/CloudTests.dll --verbosity n
`
``

Reply:
You are right, it is built for all versions but run only on 5.0. So, I added a matrix to run on others. I think Ali added other versions just to build since build version wasn't specified. So, dotnet asked the other SDKs to build for targeted frameworks.

Reply:
I turned to workflow to work on `net6.0` as before. If it's okay for you, could you resolve the items?